### DISCUSSION TITLE
RM Zorro missing Displays page in Models. (only 11 pages per model)

### DISCUSSION THREAD

**Discussion by Charlieelliott24** [EdgeTX/edgetx#2883] [Q&A] [answered]
Hey y'all!

If anyone knows a potential cause to this issue, or if I'm missing something basic/crucial, I would love to hear it! Haven't been able to find much on google. 

Running Edgetx 2.8
BF luas installed
trying to setup GPS telemetry, but can't do that without the displays page. 

Thanks!!

**Comment by pfeerick** [comment, marked as answer]
Not quite sure what the problem is - page 11 *is* the DISPLAY page, and is still present on the Zorro when running 2.8. i.e. go to model setup, go to page 11, change the value for screen 1 from none to say Nums (if you want to show number values on the first telemetry screen). Select the values you want to show on that screen. Or you can show bars if that makes sense for the data being shown. Or if you have a Lua telemetry script, you can choose Script, and then pick the script in question.

**Reply by Charlieelliott24** [reply]
Bro either I was trippin or the machine elves are out to get me.

Can't remember what I was seeing as pg11, but I swear I was positive it was not displays 😂

fuck probably just an idiot. Thank you.

**Reply by pfeerick** [reply]
Is it too late to tell you to put your glasses on? 🤭 Easily done... sometimes you can be so positive that something is missing or is there, and then when you look next time you just can't find it again... as you say, machine elves (or should that be gremlins?) 😆
